scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
 
 // focus curson in input on load
-window.onload = getInput.focus();
+window.onload = inputElement.focus();
 
 // add return key listen and call addItem function
-getInput.addEventListener('keydown', function() {
+inputElement.addEventListener('keydown', function() {
   if (event.code == 'Enter') {
     addItem();
-    getInput.value = "";
-    getInput.focus();
+    inputElement.value = "";
+    inputElement.focus();
```

Comment:
Please make up your mind. Either use a two space indentation, or four, not both.

For example, here you're using four, but in the `copy` method below, you're using two.

Reply:
Huh, thanks for drawing my attention to that, I haven't really noticed it before. 

I have Atom set to 4. I wonder why sometimes it is 2? 

I'll work on catching that more often.